auth/src/main/res/values/strings.xml

```diff
@@ -75,5 +75,10 @@
     <string name="auth_internet_unavailable_subtitle">Please check your internet connection and Try again.</string>
     <string name="auth_internet_unavailable_ok_button">Ok</string>
 
+    <!-- Login App dialog -->
+    <string name="auth_login_app_title">Login Alert</string>
```

Comment:
Please change the message as in iOS:
Title: Session unavailable
Message: Please launch Alfresco Mobile Workspace before continuing.

Reply:
Done